### ISSUE TITLE
Don't enforce port stp attribute at linux bridge if stp is false

### ISSUE THREAD

**Issue by qinqon** [nmstate/nmstate#443]
When configuring a linux bridge with stp.enable: false it's also needed to put default values of stp port attributes to nmstate to work, this is not convenient and even problematic if new version of NetworkManager is installed that has different default stp options.

Actual config needed to create a simple linux bridge
```yaml
  - name: eth1                                                                  
    type: ethernet                                                              
    state: up                                                                   
  - name: br1                                                                   
    type: linux-bridge                                                          
    state: up                                                                   
    bridge:                                                                     
      options:                                                                  
        stp:                                                                    
          enabled: false                                                        
      port:                                                                     
        - name: eth1                                                            
          stp-hairpin-mode: false                                               
          stp-path-cost: 100                                                    
          stp-priority: 32
```
Expected simplified  version

```yaml
  - name: eth1                                                                  
    type: ethernet                                                              
    state: up                                                                   
  - name: br1                                                                   
    type: linux-bridge                                                          
    state: up                                                                   
    bridge:                                                                     
      options:                                                                  
        stp:                                                                    
          enabled: false                                                        
      port:                                                                     
        - name: eth1                         
```

Event if we by default stp.enable = false we can configure it like 

```yaml
  - name: eth1                                                                  
    type: ethernet                                                              
    state: up                                                                   
  - name: br1                                                                   
    type: linux-bridge                                                          
    state: up                                                                   
    bridge:                                                                                                                   
      port:                                                                     
        - name: eth1                                      
```

**Comment by qinqon** [comment]
@maiqueb @phoracek @cathay4t

**Comment by EdDev** [comment]
@qinqon , this should have been resolved by https://github.com/nmstate/nmstate/pull/354 .
It is part of 0.0.8.

**Comment by qinqon** [comment]
It's working locally let's wait for e2e tests are kubernetes-nmstate https://github.com/nmstate/kubernetes-nmstate/pull/168

**Comment by qinqon** [comment]
Working in the k8s-nmstate PR too.

**Comment by cathay4t** [comment]
Closing as fixed in #354
